docs/features/AUTH_OAUTH_REQUIREMENTS.md

```
@@ -1,203 +1,203 @@
+                              ▼
+┌─────────────────────────────────────────────────────────────┐
+│                    Supabase (Backend)                        │
+├─────────────────────────────────────────────────────────────┤
+│  Auth Service ──► Google OAuth Provider ──► User Table      │
+└─────────────────────────────────────────────────────────────┘
+```
+
+### Components to Create/Modify
+
+#### New Components
+| Component | Path | Purpose |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)